### PR TITLE
Pull #989: Make config files kebab-case

### PR DESCRIPTION
**Pull request by stoyanK7**
From https://github.com/checkstyle/checkstyle/pull/12755#discussion_r1111293296

> I see the issue now. We are passing sevntu our XML configurations so it can run them. However, our config still references our suppression file naming and not sevntu's file naming. Sevntu still has the old name, and our configuration has the new name. Hence it is not finding the file and failing.

### CODE REVIEW

File: eclipsecs-sevntu-plugin/pom.xml [hunk 1]
```diff
@@ -29,9 +29,9 @@
     <eclipsecs.version>10.4.0-SNAPSHOT</eclipsecs.version>
     <!-- verify time version -->
     <checkstyle.version>10.7.0</checkstyle.version>
-    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</checkstyle.configLocation>
+    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle-checks.xml</checkstyle.configLocation>
```
Comment (rnveach) [inline eclipsecs-sevntu-plugin/pom.xml:32]: This change has to be delayed. `checkstyle.version` is released version, not snapshot.

File: eclipsecs-sevntu-plugin/pom.xml [hunk 2]
```diff
@@ -29,9 +29,9 @@
...
     <checkstyle.header>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java.header</checkstyle.header>
-    <checkstyle.regexp.header>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java_regexp.header</checkstyle.regexp.header>
+    <checkstyle.regexp.header>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java-regexp.header</checkstyle.regexp.header>
```
Comment (rnveach) [inline eclipsecs-sevntu-plugin/pom.xml:34]: This is ok, since it references master, which is snapshot.

File: sevntu-checks/pom.xml [hunk 3]
```diff
@@ -19,10 +19,10 @@
     <checkstyle.eclipse-cs.version>10.4</checkstyle.eclipse-cs.version>
     <!-- verify time version -->
     <checkstyle.version>10.7.0</checkstyle.version>
-    <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</checkstyle.configLocation>
-    <checkstyle.nonMain.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_non_main_files_checks.xml</checkstyle.nonMain.configLocation>
```
Comment (rnveach) [inline sevntu-checks/pom.xml:23]: Release Version. Needs revert.

File: sevntu-checkstyle-sonar-plugin/pom.xml [hunk 4]
```diff
@@ -180,7 +180,7 @@
               <goal>check</goal>
             </goals>
             <configuration>
-              <configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</configLocation>
```
Comment (rnveach) [inline sevntu-checkstyle-sonar-plugin/pom.xml:183]: Release Version. Needs revert.

**Review by rnveach — CHANGES_REQUESTED** [review]

**Comment by rnveach** [comment]
We are missing travis CI run with a lot of testing.

**Comment by romani** [comment]
I think we can merge checkstyle repo change first, make a release and update all downstream project after release. Short broken state in this repo is ok.

**Comment by rnveach** [comment]
@romani Then please tell us how to make main repo's CI green. It is red because we use same file name convention here as in main repo.

**Comment by romani** [comment]
I made a comment in checkstyle repo.

**Comment by romani** [comment]
Travis will start on next push

File: eclipsecs-sevntu-plugin/pom.xml (hunk 1 above)
Comment (stoyanK7) [reply eclipsecs-sevntu-plugin/pom.xml]: Reverted. We can open an issue for this after this PR gets merged, yes?

File: sevntu-checks/pom.xml (hunk 3 above)
Comment (stoyanK7) [reply sevntu-checks/pom.xml:23]: Reverted.

File: sevntu-checkstyle-sonar-plugin/pom.xml (hunk 4 above)
Comment (stoyanK7) [reply sevntu-checkstyle-sonar-plugin/pom.xml]: Reverted.

File: eclipsecs-sevntu-plugin/pom.xml (hunk 1 above)
Comment (rnveach) [reply eclipsecs-sevntu-plugin/pom.xml]: Yes we can.

**Comment by rnveach** [comment]
Travis is still having run issues. I am going to continue reviewing the PR.

@stoyanK7 Please ensure `mvn verify` runs without issue for `sevntu-checks` folder.

**Review by rnveach — APPROVED** [review]

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**